### ISSUE TITLE
Show/Hide info bar <i> should auto unhide when Toggling filter <f> 

### ISSUE THREAD

**Issue by clayms** [kdash-rs/kdash#446]
**Is your feature request related to a problem? Please describe.**
Info bar that is hidden with the "<i>  Show/Hide info bar "  key binding, should automatically  be unhide when Toggling the filter with the <f> key binding.  Otherwise the Filter input is also hidden.

**Comment by deepu105** [comment]
Good catch. Should be fairly simple to fix, would you like to try and fix it?

**Comment by clayms** [comment]
I won't have the time in the coming month or two.
Not a big deal for me.
Could be an easy win for anyone else that wants it though.
